server/controllers/version.controller.js

```diff
@@ -0,0 +1,15 @@
+import User from '../models/user.model'
+import { getAppVersion, getAppVersionSync, composePattern, composePatternSync } from 'appversion'
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import of 'getAppVersion'. ([javascript:S1128](https://sonarcloud.io/organizations/aumanjoa-github/rules#rule_key=javascript%3AS1128))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=chronas-api&pullRequest=32&issues=AWblUXkjSgN4uCclguYU&open=AWblUXkjSgN4uCclguYU)

[ISSUE_KEY]: AWblUXkjSgN4uCclguYU